presto-spark-base/pom.xml

```diff
@@ -126,6 +146,16 @@
             <artifactId>bootstrap</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.airlift</groupId>
+            <artifactId>http-client</artifactId>
+        </dependency>
```

Comment:
Please rebase your local branch and force push. I think this has been added by @tanjialiang so you can remove this.